site/docs/v1/tech/installation-guide/_load-testing-intro.adoc

```diff
@@ -0,0 +1,11 @@
+You can load test FusionAuth to determine if its performance will meet your calculated needs. The FusionAuth team tests the performance of FusionAuth regularly, but every situation is different and running your own load tests on your instance provides useful information.
+
+You may use your own load testing tool. Any tool which can make HTTP requests, such as gatling or jmeter, will work.
+
+There is an https://github.com/FusionAuth/fusionauth-load-tests/[open source project] which provides some useful scripts for load testing. This is a specialized library and primarily used for internal purposes, so use at your own risk. Scenarios currently supported include:
+
+* Creating tenants
+* Creating applications
+* Registering users for applications
+* Logging users in using the Login API
+* Beginning the Authorization Code grant
```

Comment:
We do actually do a full auth code grant (GET HTML, POST form, exchange auth code for access token)